### PR TITLE
miniogw: set Headers for Video Files

### PR DESCRIPTION
**Pull request by stefanbenten**

### CODE REVIEW

**Comment by kaloyan-raev** [comment]
I don't think we should merge this change to master, but just have a local build for the specific use case that requires it for now. Until we find a more generic solution.

File: miniogw/gateway.go [hunk 1]
```diff
@@ -639,6 +639,12 @@ func minioObjectInfo(bucket, etag string, object *uplink.Object) minio.ObjectInf
 	if etag == "" {
 		etag = object.Custom["s3:etag"]
 	}
+	// Set Headers to properly return videofiles
+	if needsVideoHeaders(object.Key) {
```
Comment (egonelbre) [inline miniogw/gateway.go:643]: Why doesn't the original upload contain the correct headers?

I suspect we are missing a ton of video extensions.

**Comment by stefanbenten** [comment]
Yeah, its mainly to have it reproducable for the customer..

File: miniogw/gateway.go (hunk 1 above)
Comment (stefanbenten) [reply miniogw/gateway.go:643]: We are, this is more or less a quick fix as we do not allow metadata editing.

**Pull request closed without merging**